app.js

```diff
@@ -0,0 +1,165 @@
+var hours = ['6am', '7am', '8am', '9am', '10am', '11am', '12pm', '1pm', '2pm', '3pm', '4pm', '5pm', '6pm', '7pm', '8pm'];
+
+var pike = {
+  minCust: 23,
+  maxCust: 65,
+  avgCookies: 6.3,
+  hourlySales: []
+};
```

Comment:
Interesting that you didn't include an array that held onto the number of customers each hour. Nice work.